project/Dependencies.scala

```diff
@@ -196,42 +232,46 @@ object Dependencies {
 
   val clusterTools = l ++= Seq(Test.junit, Test.scalatest.value)
 
-  val clusterSharding = l ++= Seq(
-        Provided.levelDBmultiJVM,
-        Provided.levelDBNative,
-        Test.junit,
-        Test.scalatest.value,
-        Test.commonsIo)
+  val clusterSharding = l ++= Provided.levelDBmultiJVM ++
+  Provided.levelDBNative ++
+  Seq(
+      Test.junit,
+      Test.scalatest.value,
+      Test.commonsIo
+    )
 
-  val clusterMetrics = l ++= Seq(Provided.sigarLoader, Test.slf4jJul, Test.slf4jLog4j, Test.logback, Test.mockito)
+  val clusterMetrics = l ++= Provided.sigarLoader ++ Seq(Test.slf4jJul, Test.slf4jLog4j, Test.logback, Test.mockito)
 
   val distributedData = l ++= Seq(lmdb, Test.junit, Test.scalatest.value)
 
   val slf4j = l ++= Seq(slf4jApi, Test.logback)
 
-  val persistence = l ++= Seq(
-        Provided.levelDB,
-        Provided.levelDBNative,
+  val persistence = l ++=
+    Provided.levelDB ++
+    Provided.levelDBNative ++
+    Seq(
         Test.scalatest.value,
         Test.junit,
         Test.commonsIo,
         Test.commonsCodec,
-        Test.scalaXml)
+        Test.scalaXml
+      )
 
-  val persistenceQuery = l ++= Seq(
+  val persistenceQuery = l ++=
+    Provided.levelDB ++
+    Provided.levelDBNative ++
+    Seq(
         Test.scalatest.value,
         Test.junit,
         Test.commonsIo,
-        Provided.levelDB,
-        Provided.levelDBNative)
+      )
 
-  val persistenceTck = l ++= Seq(
-        Test.scalatest.value.withConfigurations(Some("compile")),
-        Test.junit.withConfigurations(Some("compile")),
-        Provided.levelDB,
-        Provided.levelDBNative)
+  val persistenceTck = l ++=
+    Provided.levelDB ++
+    Provided.levelDBNative ++
+    Seq(Compile.scalatest.value, Compile.junit)
```

Comment:
Here (and elsewhere applicable) I'd list starting with `Compile` and moving to `Provided` given below you do that same progression, to keep it consistent, compile -> provided.

Reply:
Will fix